srf_generation/source_parameter_generation/common.py

```diff
@@ -133,3 +133,14 @@ def get_depth_property(target_depths, vel_mod_1d_layers, property_name):
         np.asarray(target_depths).round(5), vel_mod_1d_layers["depth"].cumsum().round(5)
     )
     return np.asarray(vel_mod_1d_layers[property_name].iloc[binned_depths])
+
+
+def write_asperites(asperities_dict, realisation_file_name):
```

Comment:
docstring saying what it does and return?

Just because the return value isn't obvious.

Maybe it's better passing a filename into this function, rather than the rel_file_name?